Synview-backend/deps.ts

```diff
@@ -0,0 +1,10 @@
+export { serve } from "https://deno.land/std@0.140.0/http/server.ts";
+export { withAccelerate } from "npm:@prisma/extension-accelerate";
+export { PrismaClient } from './generated/prisma/client.ts';
```

Comment:
I am not a great fan of having a deps file like this - but don't have a super strong opinion on it. Did you see other folks doing this in Codepath? 

My main complaint with it is that it makes it harder when reading other files to see where exactly some dependency comes from (ie. Application)